### DISCUSSION TITLE
Working Sets

### DISCUSSION THREAD

**Discussion by TexasAthletics** [dbeaver/dbeaver#35039] [Q&A] [answered]
When you create a new project in dbeaver community 24.1.3, there's an option to add the new project to a working set.  I cant find any documentation, blog posts, youtube videos, google results, chatgpt responses, etc that describes what a 'Working Set' is.  I also cant seem to figure out any sort of hierarchical aspect of organizing the projects via these 'Working Sets'  Can anyone clarify this? 
![working sets](https://github.com/user-attachments/assets/620b0d05-63be-47fe-99b0-55f86b160ac7)

**Comment by TB-HunBoka** [comment, marked as answer]
Developers will know better, but for me it looks like an inherited function from Eclipse.
https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.platform.doc.user%2Fconcepts%2Fcworkset.htm
Search for "Eclipse working sets" instead. It will give you more results about what it does.

**Comment by GTonehour** [comment]
The only DBeaver documentation page including the term "working set" is https://dbeaver.com/docs/dbeaver/Toolbar-Customization/, which doesn't explain the concept. I understand it could be similar to Eclipse's notion, but it would be interesting to have a source from the DBeaver team.
